src/trip/components/tripsTable.jsx

```diff
@@ -0,0 +1,66 @@
+import React, { useEffect, useState } from 'react'
+import { getCredentials, register, unregister } from '../../login/store'
+
+import { Table, Button } from 'semantic-ui-react'
+import dateFormat from '../../utils/dateFormat'
+
+const TripsTable = ({ trips, isBack }) => {
+    const [title] = useState(isBack ? 'Trajet retour' : 'Trajet Aller')
+    const [credentials, setCredentials] = useState()
+    const [connectedUser, setConnectedUser] = useState()
```

Comment:
Do you really need that connectedUser thing, couldn't you just use credentials.login everywhere ? Or is it that if you have only a single useEffect it is also called at updates ?

Reply:
Same here, PR should have been a draft PR, i was only testing stuff :)

You are right, no need to keep the connectedUser state